tests/reporter/discord-reporter.js

```
@@ -0,0 +1,114 @@
+}
+
+function truncate(str, max) {
+  if (!str) return '';
+  return str.length <= max ? str : `${str.slice(0, max - 1)}…`;
+}
+
+function stripAnsiCodes(str) {
+  if (!str) return '';
+  // Remove ANSI escape codes (colors, formatting, etc.)
+  // eslint-disable-next-line no-control-regex
+  return str.replace(/\x1b\[[0-9;]*m/g, '');
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)